atomic_reactor/schemas/fetch-artifacts-nvr.json

```diff
@@ -0,0 +1,56 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+
+  "title": "Artifacts to be fetched from Koji builds.",
```

Comment:
We should be consistent (with other files in schemas/) in whether we use a full-stop in the title and descriptions. Currently config.json doesn't use them.